daemon/algod/api/server/v2/handlers.go

```diff
@@ -1662,3 +1660,8 @@ func (v2 *Handlers) TealDisassemble(ctx echo.Context) error {
 	}
 	return ctx.JSON(http.StatusOK, response)
 }
+
+// ExperimentalCheck is only available when EnabledExperimentalAPI is true
+func (v2 *Handlers) ExperimentalCheck(ctx echo.Context) error {
```

Comment:
IsExperimentalEnabled?

Reply:
I agree the function name is not great but this has to match the route's oas2 `operationId`.

And since we have other endpoints like `HealthCheck` I think this is consistent. Willing to change if there is strong pushback though.